build_helpers/library_config.rs

```diff
@@ -319,27 +320,63 @@ impl LibraryConfig {
     ) -> Result<(), Error> {
         let lib_name = OsString::from(&lib_name);
         let library = self.cfg.probe(lib_name.to_str().unwrap())?;
+
+        println!("Found pkg config dependencies for {lib_name:?}:");
+
         for name in &library.libs {
             let lib = Library::new(name, &lib_name, self);
+            println!("    link lib {n:?}", n = lib.pkg_name);
```

Comment:
Extra white spaces can be removed from `println` in this file ( unless intentionally kept).

Reply:
It's an intention indent, to make the output a bit easier to follow.